docs/pages/includes/plugins/rbac-with-friendly-name.mdx

```diff
@@ -0,0 +1,50 @@
+Teleport's Access Request plugins authenticate to your Teleport cluster as a
+user with permissions to list and read Access Requests. This way, plugins can
+retrieve Access Requests from the Teleport Auth Service and present them to
+reviewers.
+
+Define a user and role called `access-plugin` by adding the following content to
+a file called `access-plugin.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: access-plugin
+spec:
+  allow:
+    rules:
+      - resources: ['access_request']
+        verbs: ['list', 'read']
+      - resources: ['access_plugin_data']
+        verbs: ['update']
+      # Optional: To display user-friendly names in resource-based access
+      # requests instead of resource IDs, the plugin also needs permission
+      # to list the resources being requested. Add applicable allow rules
+      # and labels.
+      - resources: ['node', 'app', 'db', 'kube_cluster']
+        verbs: ['list', 'read']
+    node_labels:
+      '*': '*'
```

Comment:
Would using `preview_as_roles` be more secure and finely scoped here?

Reply:
I think if we used `preview_as_roles`, the role the plugin would preview as would need these same permissions, so I don't think we'd be gaining any security.

Reply:
Well here we have `'*': '*'` in a bunch of places, which means the access plugin role actually has permissions to access just about every resource in the cluster. That's a pretty high level of permission for something that actually only needs to work with access requests.

I think `preview_as_roles` would give the plugin enough permissions to look up the resource names without giving it permission to access them.

Reply:
Looks better now, assuming you validated that it works the same :-)

Reply:
I checked and it does.